src/app/Pages/Blog.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react'
+import {
+   PageSection, Bullseye, Text, TextContent, TextVariants,
+  Card, CardTitle, CardBody, CardFooter
+} from '@patternfly/react-core'
+import { useDocumentTitle } from '@app/utils/useDocumentTitle'
+import Footer from '@app/components/Footer'
+import post11png from '@app/bgimages/post1-1.png'
+
+const Blog: React.FunctionComponent<{title: string}> = ({title}) => {
+
+  useDocumentTitle(title)
+  return (
+    <div>
+      <PageSection isFilled={true}>
+      <Bullseye>
+        <Card
+          style={{
+            width: 800,
+            display: 'flex',
+            justifyContent: 'center',
+          }}>
+          <CardTitle>
+            <p>The new Cloud Image Directory is now available</p>
+            <Text component={TextVariants.small}>Author: The Cloud Experience Team</Text>
```

Comment:
Cloud Experience Team is also represented as Cloud Experience team later in the blog. I am fine with either one, but we should be consistent.

Reply:
@FKolwa, Thanks for writing this blog. I copyedited it. Please let me know if you disagree with any of my suggestions or if you have any questions.